optuna/integration/mlflow.py

```diff
@@ -75,14 +75,30 @@ def objective(trial):
             Name of the metric. Since the metric itself is just a number,
             `metric_name` can be used to give it a name. So you know later
             if it was roc-auc or accuracy.
+        nest_trials:
+            Flag indicating whether or not trials should be logged as
+            nested runs. This is often helpful for aggregating trials
+            to a particular study, under a given experiment.
+        tag_study_user_attrs:
+            Flag indicating whether or not to add the study's user attrs
+            to the mlflow trial as tags. Please note that when this flag is
+            set, key value pairs in study.user_attrs will supersede existing tags.
```

Comment:
Okay, so the logic is actually that the study user attributes are superseded too, so the explanation is somewhat misleading. I think we should either update the docs, or the logic (I'm fine with either).